packages/mobile/src/components/now-playing-drawer/NowPlayingDrawer.tsx

```diff
@@ -41,28 +41,37 @@ import { TitleBar } from './TitleBar'
 import { TrackInfo } from './TrackInfo'
 import { PLAY_BAR_HEIGHT } from './constants'
 
-const combinedBottomAreaHeight = BOTTOM_BAR_HEIGHT + PLAY_BAR_HEIGHT
-
 const STATUS_BAR_FADE_CUTOFF = 0.6
 const SKIP_DURATION_SEC = 15
+// If the top screen inset is greater than this,
+// the status bar will be hidden when the drawer is open
+const INSET_STATUS_BAR_HIDE_THRESHOLD = 20
```

Comment:
cool, so is this like "hide for phones without notches"?

Reply:
Yeah, on small screens (without a notch) the drag would get interrupted because the status bar added/removed extra space at the top